lib/message.rb

```diff
@@ -0,0 +1,20 @@
+require './lib/morse_translator'
+require 'pry'
+class Message
+  attr_reader :text,
+              :morse_translator
+
+  def initialize(text, morse_translator)
+    @text             = text
+    @morse_translator = morse_translator
+  end
+
+  def translation
+    translation = ""
+    @text.each_char do |char|
+      translation << @morse_translator.dictionary[char.downcase]
+    end
+    translation
```

Comment:
This looks like it does the thing, but instead of accessing the `dictionary` directly in MorseTranslator, use the method `translate` that the class provides you. You're already doing that work over in that class. No need to replicate it here.